libs/community/langchain_community/graph_vectorstores/extractors/hierarchy_link_extractor.py

```diff
@@ -0,0 +1,74 @@
+from typing import Callable, List, Set
+
+from langchain_core.documents import Document
+from langchain_core.graph_vectorstores.links import Link
+
+from langchain_community.graph_vectorstores.extractors.link_extractor import (
+    LinkExtractor,
+)
+from langchain_community.graph_vectorstores.extractors.link_extractor_adapter import (
+    LinkExtractorAdapter,
+)
+
+# TypeAlias is not available in Python 2.9, we can't use that or the newer `type`.
```

Comment:
Can remove this comment since we're in >= python 3.8

Reply:
Updated the comment. This was supposed to be `3.9`, indictaing why it doesn't use the new `HierarchyInput: TypeAlias = List[str]` style.